core/ledger/ledger.go

```diff
@@ -0,0 +1,187 @@
+package ledger
+
+import (
+	"context"
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"time"
+
+	ledgerPb "github.com/TRON-US/go-btfs/core/ledger/pb"
+
+	"github.com/gogo/protobuf/proto"
+	logging "github.com/ipfs/go-log"
+	ic "github.com/libp2p/go-libp2p-core/crypto"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+)
+
+var log = logging.Logger("ledger")
+var (
+	ledgerAddr = "ledger-nginx.bt.co:443"
+	certFile   = `-----BEGIN CERTIFICATE-----
```

Comment:
do we want put the certificate here in the open source repo?

Reply:
It will be removed soon when we switch to escrow. But we have to put it here for the demo code to connect to dev ledger. grpc https requirement.